blocksuite/affine/blocks/surface/src/renderer/dom-renderer.ts

```diff
@@ -483,4 +453,256 @@ export class DomRenderer {
     this._overlays.delete(overlay);
     this.refresh();
   }
+
+  /**
+   * Mark a specific element as dirty for incremental updates
+   * @param elementId - The ID of the element to mark as dirty
+   * @param updateType - The type of update (optional, defaults to ELEMENT_UPDATED)
+   */
+  markElementDirty(
```

Comment:
it is very likely the public API will be called with a modified `this` closure. Make sure the `this` is pointed to the correct instance by changing it to an arrow method.